lib/travis/api/v3/queries/log.rb

```diff
@@ -9,6 +9,8 @@ def find(job)
       #if the log has been archived, go to s3
       if log.archived_at
         content = fetch.get(prefix).try(:body)
```

Comment:
The second if for the existence of content is not needed. Can the fetch.get return nil or will it error before coming back?